composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "minimum-stability": "stable",
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
+        "phpunit/phpunit": "4.6.*",
```

Comment:
> I prefer not to bother composer with PHPUnit, similar to composer self-update.

That;s not the same though. You're installing phpunit with composer on travis, then not using it?